components/web/php/php-7_0/php70-ext-pgsql.p5m

```diff
@@ -23,5 +23,5 @@ set name=info.classification \
 license LICENSE license="PHP56 License"
 
 file pgsql.ini path=/etc/php/7.0/conf.d/pgsql.ini preserve=true
-file path=/usr/php/7.0/extensions/$(MACH64)/pgsql.so
-file path=/usr/php/7.0/extensions/pgsql.so
+file path=/usr/php/7.0/extensions/$(MACH64)/pgsql.so pkg.depend.bypass-generate=.*
```

Comment:
Do you have postgresql94 installed?

Reply:
Yes is that the problem? I know that in some of my zones  python ffi is broken so I thought it might be that problem (to which I have not found a solution)

Reply:
No. Component is looking for $(PG_BINDIR)/pg_config, and PG_BINDIR is /usr/postgres/9.4/bin or /usr/postgres/9.4/bin/amd64

Reply:
Ah yes Zabbix is looking for pg_config and that works. pkgsend dependency resolv however complains. And I guess that is Python FFI.

Reply:
How exactly does it complain?

Reply:
@Toasterson works here without pkg.depend.bypass-generate=.* . Also you have removed 32-bit extension

Reply:
@pyhalov Look below the conversation :)

Reply:
This is what I get. As said I believe it is my build zone. However I have not found a way to fix it and it probably will happen again if I recreate the zone.
```
/root/userland/components/web/php/php-7_0/build/manifest-i386-php70-ext-pgsql.depend has unresolved dependency '
    depend type=require fmri=__TBD \
        pkg.debug.depend.reason=usr/php/7.0/extensions/pgsql.so \
        pkg.debug.depend.type=elf \
        pkg.debug.depend.fullpath=lib/libpq.so.5 \
        pkg.debug.depend.fullpath=usr/gcc/6/lib/libpq.so.5 \
        pkg.debug.depend.fullpath=usr/mariadb/10.1/lib/libpq.so.5 \
        pkg.debug.depend.fullpath=usr/postgres/11/lib/amd64/libpq.so.5'.
/root/userland/components/web/php/php-7_0/build/manifest-i386-php70-ext-pgsql.depend has unresolved dependency '
    depend type=require fmri=__TBD \
        pkg.debug.depend.reason=usr/php/7.0/extensions/amd64/pgsql.so \
        pkg.debug.depend.type=elf \
        pkg.debug.depend.fullpath=lib/64/libpq.so.5 \
        pkg.debug.depend.fullpath=usr/gcc/6/lib/amd64/libpq.so.5 \
        pkg.debug.depend.fullpath=usr/mariadb/10.1/lib/amd64/libpq.so.5 \
        pkg.debug.depend.fullpath=usr/postgres/11/lib/amd64/libpq.so.5'.
gmake: *** [/root/userland/make-rules/ips.mk:439: /root/userland/components/web/php/php-7_0/build/.resolved-i386] Error 1
```

Reply:
This means it somehow finds and uses pg_config from PostgreSQL 11, not one from PG_BINDIR. What does gmake print-PG_BINDIR say? Do you have /usr/postgres/9.4/bin/pg_config ?

Reply:
Ah ok I get the following. ANd Yes I have postgres11 installed. Should I try with removed postgres11?
```
gmake print-PG_BINDIR
PG_BINDIR=/usr/postgres/9.4/bin (origin: file, flavor: recursive)
```

Reply:
No